src/navigation/optOutFooter.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+// this provides an opt-out link in the footer for the compulsory opt-in phase of Next
+module.exports = function (items, currentUrl) {
+	const optOutLink = {
+		name: 'Old FT.com',
+		href: `/opt-out-confirm?location=${encodeURIComponent(currentUrl)}`
+	};
+	const supportBlock = items.find(i => {
+		return i.title === 'Support';
+	});
+	if (supportBlock) {
+		supportBlock.optOutLink = optOutLink;
```

Comment:
you can see in the n-ui footer pr, if this item is present it outputs it to the top of the list of links if certain conditions are true